src/redux/sagas/extensions/extensionInstall.ts

```diff
@@ -49,6 +72,91 @@ export function* extensionInstall({
   return null;
 }
 
+function* handleMultiSigInstall(colonyAddress: string, metaId: string) {
+  const batchKey = 'setInitialMultiSigRoles';
+
+  const apolloClient = getContext(ContextModule.ApolloClient);
+  const colonyManager = yield getColonyManager();
+
+  const colonyClient = yield colonyManager.getClient(
+    ClientType.ColonyClient,
+    colonyAddress,
+  );
+
+  const childSkillIndex = yield call(
+    getChildIndex,
+    colonyClient.networkClient,
+    colonyClient,
+    Id.RootDomain,
+    Id.RootDomain,
+  );
```

Comment:
This should be fetching the index from the DB as opposed to chain to save on client-side network calls